pkg/algorithms/algorithm.py

```diff
@@ -113,10 +113,10 @@ def group_change_points_by_time(
         changes: List[ChangePoint] = []
```

Comment:
nit-pick: This file doesn't seem to be having any changes in it. can we just discard this from this PR?